python/test/testutil.py

```diff
@@ -25,56 +25,58 @@
 import sys
 import tempfile
 
+class TestSetupMixin(object):
 
-TEST_APP_ID = 'my-app-id'
-TEST_VERSION_ID = 'my-version.1234'
+  test_app_id = 'my-app-id'
+  test_version_id = 'my-version.1234'
 
-# Assign the application ID up front here so we can create db.Key instances
-# before doing any other test setup.
-os.environ['APPLICATION_ID'] = TEST_APP_ID
-os.environ['CURRENT_VERSION_ID'] = TEST_VERSION_ID
-os.environ['HTTP_HOST'] = '%s.appspot.com' % TEST_APP_ID
-os.environ['DEFAULT_VERSION_HOSTNAME'] = os.environ['HTTP_HOST']
-os.environ['CURRENT_MODULE_ID'] = 'foo-module'
+  os.environ['APPLICATION_ID'] = test_app_id
```

Comment:
Maybe this should move to the setUp as well (if we are really changing TEST_APP_ID).